macros/models/_sector/nft/platforms/element_v1_base_trades.sql

```diff
@@ -0,0 +1,123 @@
+-- Element NFT trades (re-usable macro for all chains)
+{% macro element_v1_base_trades(erc721_sell_order_filled, erc721_buy_order_filled, erc1155_sell_order_filled, erc1155_buy_order_filled) %}
+
+
+SELECT
+  DATE_TRUNC('day', evt_block_time) AS block_date,
+  evt_block_time AS block_time,
+  evt_block_number AS block_number,
+  'Buy' AS trade_category,
+  'secondary' AS trade_type,
+  erc721Token AS nft_contract_address,
+  cast(erc721TokenId as uint256) AS nft_token_id,
+  uint256 '1' AS nft_amount,
+  taker AS buyer,
+  maker AS seller,
+  cast(erc20TokenAmount AS UINT256) AS price_raw,
+  CASE
+    WHEN erc20Token = 0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee THEN 0x0000000000000000000000000000000000000000
+    ELSE erc20Token
+  END AS currency_contract,
+  cast(0 AS UINT256) AS platform_fee_amount_raw,
```

Comment:
nit: uint256 '0'

Reply:
There are a few of these and also `cast(1 as uint256)`.

Reply:
Replacing these all post migration. :heavy_check_mark: